src/controller.coffee

```diff
@@ -140,6 +140,8 @@ define ['droplet-helper',
   exports.Editor = class Editor
     constructor: (@wrapperElement, @options) ->
       @paletteGroups = @options.palette
+      @willShowPaletteWithText = @options.willShowPaletteWithText ? false
```

Comment:
I found my brain having trouble parsing this option name (i.e. I was reading as will there be text in the palette instead of will there be a palette in tell mode). What about willShowPaletteInTextMode or showPaletteInTextMode?